charts/pega/README.md

```diff
@@ -124,6 +122,9 @@ docker:
     url: "YOUR_DOCKER_REGISTRY"
     username: "YOUR_DOCKER_REGISTRY_USERNAME"
     password: "YOUR_DOCKER_REGISTRY_PASSWORD"
+  pega:
+    image: "pegasystems/pega"
+    imagePullPolicy: "Always"
```

Comment:
Why do we want `Always` as the default? It will make an unnecessary attempt to download the same image that may already be present locally.

Reply:
I don't think this PR changes it to always by default, thats just the example.  The README incorrectly says its Always by default which is what I mentioned i nthe original issue

WE (SQuID) need this beecause we are constantly refreshing tha specific packed web image (not latest  tag) for the helm SDEs

Reply:
SO Dave C has kindly provided us access to this value for our use case

Reply:
II also suspect that if a customer is using a non-latest tag for their image, they will want to use this option to make sure the get the latest tagged version with all of the security updates (since we periodically refresh tagged versions with security updates)

Reply:
Correct that this is just the readme.  The default is not having an `imagePullPolicy` set.